internal/pkg/template/templates/workloads/services/lb-web/cf.yml

```diff
@@ -169,7 +169,64 @@ Resources:
 
 {{- if .ALBEnabled}}
 {{include "alb" . | indent 2}}
+
+  CustomResourceRole:
```

Comment:
At a glance the `CustomResourceRole` for lbws and backend service seem to be the same, am I missing something or can we potentially move the role to a partial?

Reply:
Previously, the `CustomResourceRole` was in the `alb` partial, but the issue with it there is that the backend service needs it if autoscaling is enabled, regardless of if it needs an ALB or not. I considered putting it in it's own partial, but think the best long term solution (since the worker service uses it as well) is to refactor out the `CustomResourceRole` everywhere and create a lambda-specific role instead. 

I've started that work [here](https://github.com/aws/copilot-cli/compare/mainline...dannyrandall:fix/nlb-autoscaling), but want to finish the internal ALB stuff before doing that.

Thoughts? You're absolutely right that the role is the exact same, so it's something that could be solved sooner by putting it in it's own partial.